qa/util/src/main/java/io/camunda/qa/util/cluster/TestStandaloneCamunda.java

```diff
@@ -261,13 +236,6 @@ public TestStandaloneCamunda withExporter(final String id, final Consumer<Export
     return this;
   }
 
-  /**
-   * Sets the broker's working directory, aka its data directory. If a path is given, the broker
-   * will not delete it on shutdown.
-   *
-   * @param directory path to the broker's root data directory
-   * @return itself for chaining
-   */
```

Comment:
❌ Can you explain why you removed all comments in this class?

Reply:
Removed by accident in a refactor, going to readd it.